packages/create-react-admin/src/cli.tsx

```diff
@@ -55,17 +112,21 @@ const cli = meow(
 if (cli.flags.h) {
     cli.showHelp();
 } else {
-    const dataProvider = cli.flags.basic ? 'none' : cli.flags.dataProvider;
-    const authProvider = cli.flags.basic ? 'none' : cli.flags.authProvider;
-    const install = cli.flags.basic ? 'npm' : cli.flags.install;
-    const resources =
-        cli.flags.basic || cli.flags.resource.includes('skip')
-            ? []
-            : cli.flags.resource;
+    const name = cli.input.length > 0 ? cli.input[0].trim() : undefined;
+    if (!name && !cli.flags.interactive) {
+        console.error(
+            'Please provide a name for your admin application: create-admin-app <name>'
```

Comment:
same here

Reply:
No, this one is actually required if not interactive

Reply:
Yeah but what I meant is that the message still mentions `create-admin-app` which is incorrect :grimacing: